cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -392,8 +392,8 @@ func NewControllerInitializers(loopMode ControllerLoopMode) map[string]InitFunc
 	controllers["garbagecollector"] = startGarbageCollectorController
 	controllers["daemonset"] = startDaemonSetController
 	controllers["job"] = startJobController
-	//controllers["deployment"] = startDeploymentController
-	//controllers["replicaset"] = startReplicaSetController
+	controllers["deployment"] = startDeploymentController
+	controllers["replicaset"] = startReplicaSetController
```

Comment:
this is uncommented because workload-controller-manager is disabled in this setup.